docs/website/content/v0.4/en/guides/getting-started/firecracker.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Firecracker
+---
+
+In this guide we will create a Kubernetes cluster using Firecracker.
+
+## Requirements
+
+- KVM
+- firecracker (v0.21.0 or higher)
+- CNI installed to `/opt/cni` (`CAP_SYS_ADMIN` and `CAP_NET_ADMIN` capabilities are required)
+- iptables
+
+## Create the Cluster
+
+```bash
+talosctl cluster create --wait --provisioner firecracker
+```
+
+Once the above finishes successfully, your talosconfig(`~/.talos/config`) will be configured to point to the new cluster.
+
+> Note: You must run the above as `root`.
+
+## Retrieve and Configure the `kubeconfig`
+
+```bash
+talosctl kubeconfig .
+kubectl --kubeconfig kubeconfig config set-cluster talos-default --server https://127.0.0.1:6443
+```
+
+## Using the Cluster
+
+Once the cluster is available, you can make use of `talosctl` and `kubectl` to interact with the cluster.
+For example, to view current running containers, run `talosctl containers` for a list of containers in the `system` namespace, or `talosctl containers -k` for the `k8s.io` namespace.
+To view the logs of a container, use `talosctl logs <container>` or `talosctl logs -k <container>`.
+
+## Cleaning Up
```

Comment:
we could also probably add that resources are created in `~/.talos/clusters/talos-default/...`, this place has all the console logs, etc.

and in case if host is rebooted, these files will be left behind, so they might need to be cleaned up